app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/ide.tsx

```diff
@@ -0,0 +1,74 @@
+/** @file  */
+import * as react from "react";
+
+import * as service from "../service";
+
+const IDE_CDN_URL = "https://ensocdn.s3.us-west-1.amazonaws.com/ide";
+
+interface Props {
+  project: service.Project;
+  backendService: service.Backend;
+}
+
+declare global {
+  const enso: { main: (config: object) => void; };
+}
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const Ide = ({ project, backendService }: Props) => {
+  react.useEffect(() => {
+    void (async () => {
+      const ideVersions = await backendService.listVersions({
+        versionType: service.VersionType.ide,
+        default: true,
+      });
+      /* TODO: make latest nightlies configurable again */
+      /* TODO: reload style.css and index.js.gz when opening a different version */
```

Comment:
todo left in the code: Do one of the following points:
1. Fix it in this PR.
2. Or, chat with your Project Leader and agree that it will be fixed as a separate PR. In this case, an issue should be created, described and linked in the TODO comment. You should also annotate the TODO comment with you initials. Again, see code style guide to learn more.